create_MyString_file.py

```diff
@@ -0,0 +1,60 @@
+class MyString:
+    def __init__(self,s):
+        self.string = s
+        self.length = MyString.len(self)
+
+    def __str__(self):
+        return self.string
+
+    def len(self):
+        count = 0
+        for char in self.string:
+            count += 1
+        return count
+
+    def add_string(self, other):
+        return '{}{}'.format(self.string,other.string)
+
+    def substring(self,start,end):
+        substr = ''
+        for i in range(start,end):
+            if i >= self.length:
+                break
+            substr += self.string[i]
```

Comment:
may I ask for the difference between implementing add_string method and overriding + operator? I don't understand it clearly because in the previous review when I overrode + operator as the link you gave me, you said I was not allowed to use "+" (maybe I do not understand how to override it). 

And just a quick wonder, how can I type  "add_string" or "+" in block form like yours?